### ISSUE TITLE
Conversion workers count is hard coded to 1

### ISSUE THREAD

**Issue by Sn0wFox** [ONLYOFFICE/DocumentServer#2594]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Docker

### Version information

8.0.0

### Expected Behavior

Converter workers from https://hub.docker.com/r/onlyoffice/docs-converter-de should scale with available CPUs and configuration option `FileConverter.converter.maxprocesscount`

### Actual Behavior

Only 1 converter worker is available

### Reproduction Steps

1. Run https://hub.docker.com/r/onlyoffice/docs-converter-de in suitable environment
2. Check the logs:

```
[WARN] [docId] [userId] nodeJS - num of CPUs: 48; availableParallelism: undefined
[WARN] [docId] [userId] nodeJS - update cluster with 1 workers
[WARN] [docId] [userId] nodeJS - worker 19 started.
```

Do the same with version 7.2.2.56 for example and you will see that 48 workers are created.

### Additional information

[This commit](https://github.com/ONLYOFFICE/server/commit/e3ecb2ac7f0cc02f53e3dc73fdc92b96b84f6fde#diff-449913392307f410409fce37e8ccf88a618e16baa82d6b5c1ef35e98bd4818d7L54) removed an if statement about multi-tenancy. Now the worker count is computed like so:

```js
workersCount = Math.min(licenseInfo.count, workersCount);
```

With `licenseInfo.count` being [hard coded to 1](https://github.com/ONLYOFFICE/server/blob/master/Common/sources/license.js#L45). I don't see anything overriding it with a value coming from the actual license, but even if that was the case the license I have doesn't have this `count` field and never had.

**Comment by Sn0wFox** [comment]
https://github.com/ONLYOFFICE/server/blob/master/FileConverter/sources/convertermaster.js#L56

The number of workers is always 1. Setting [`FileConverter.converter.maxprocesscount`](https://helpcenter.onlyoffice.com/installation/docs-developer-configuring.aspx#FileConverter-converter-maxprocesscount) has no effect because the worker count is always overridden by the line I already linked, which always outputs 1.

Considering there is an option that doesn't work, I'd argue it's bug regardless of your container approach.

**Comment by igwyd** [comment]
The image here https://hub.docker.com/r/onlyoffice/docs-converter-de is already builded with a multiplier of 0.001 for the ```FileConverter.converter.maxprocesscount```, as i write in my previous post. Of course you can't change it in the config because this is for the k8s. Anyway if you stit want it, take source code and build image with your specific settings https://github.com/ONLYOFFICE/Docker-Docs/tree/master. 
By the way, could you describe your deployment?

**Comment by Sn0wFox** [comment]
The deployment is the latest version of your helm chart https://github.com/ONLYOFFICE/Kubernetes-Docs with HA setup for the third party apps (redis, rabbit, postgres). During my tests, upgrading from app version 7.2.x heavily slowed down calls to the conversion API (from average 2s/conversion to 40s+) under load because all converter pods were using a single worker as I described. I had to adjust resources/scaling to maintain a reasonable conversion speed in production.

I will take a look at the custom image build as you suggested. May I ask if all options listed at https://helpcenter.onlyoffice.com/installation/docs-developer-configuring.aspx should be set during image build ? Or could some still be used at runtime ?
